schemas/1.5/dbt_project-1.5.json

```diff
@@ -818,6 +830,16 @@
         "additionalProperties": false
       }
     },
+    "grants": {
+      "type": "object",
+      "description": "flexible grant config. accepts any string key, expects a string or list of strings value",
```

Comment:
The various IDEs should take responsibility for saying what the type is. The description would be better if it was what the strings actually are (what is granted, to whom it is granted)